projects/igniteui-angular/src/lib/core/styles/themes/_core.scss

```diff
@@ -70,4 +71,5 @@
 
 @mixin igx-core() {
     @include igx-typography();
+    @include igx-print-layout;
```

Comment:
This should be a function call. Also should it always get included?

Reply:
@kdinev This is not a function, it's a mixin. 
@desig9stein However, the linter will throw because of this rule "scss/at-mixin-argumentless-call-parentheses": "always". So include it with parentheses.

Reply:
Since this is a mixin and we don't expect any params it's not necessary to be invoked like a function, however, we have a Stylelint rule that forces us to add one, so I will add it :). I spoke with @simeonoff and we decide to make it optional but it will be ON by default.